cores/arduino/wiring.h

```diff
@@ -52,4 +53,7 @@
 #define clockCyclesToMicroseconds(a) ( ((a) * 1000L) / (SystemCoreClock / 1000L) )
 #define microsecondsToClockCycles(a) ( (a) * (SystemCoreClock / 1000000L) )
 
+#define va_start(v,l)	__builtin_va_start(v,l)
```

Comment:
Those lines are not needed if you include also stdio.h.
Anyway, I will investigate to include them using wiring_private.h to be aligned with Arduino as mentioned in #240.